src/k8s/pkg/k8sd/setup/kubelet.go

```diff
@@ -81,7 +81,7 @@ func kubelet(snap snap.Snap, hostname string, nodeIP net.IP, clusterDNS string,
 		args["--cluster-domain"] = clusterDomain
 	}
 	if nodeIP != nil && !nodeIP.IsLoopback() {
-		args["--node-ip"] = utils.ToIPString(nodeIP)
+		args["--node-ip"] = nodeIP.String()
```

Comment:
drive-by fix: brackets around the IPv6 are only required/accepted if combined with a port.

Reply:
Maybe we need to document this somewhere. I personally totally forgot about out brackets convention.

Reply:
This is not our convention but the actual IP standard.